include/aws/http/request_response.h

```diff
@@ -414,6 +414,18 @@ int aws_http_headers_get(
     struct aws_byte_cursor name,
     struct aws_byte_cursor *out_value);
 
+/**
+ * Test if header name exists or not in headers
+ */
+AWS_HTTP_API
+bool aws_http_has_header_in_headers(const struct aws_http_headers *headers, struct aws_byte_cursor name);
```

Comment:
naming suggestion: `aws_http_headers_has()`

fits in with the existing `aws_http_headers_add()` and `aws_http_headers_get()`